freeipa/src/main/java/com/sequenceiq/freeipa/entity/util/ListToString.java

```diff
@@ -0,0 +1,31 @@
+package com.sequenceiq.freeipa.entity.util;
+
+import java.io.IOException;
+import java.util.List;
+
+import javax.persistence.AttributeConverter;
+
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.sequenceiq.cloudbreak.common.json.Json;
+
+public abstract class ListToString<U> implements AttributeConverter<List<U>, String> {
```

Comment:
if you would create sth like `JsonStringSetUtils` then you could use `JsonToString` to store the data. See it in action here: https://github.com/hortonworks/cloudbreak/blob/95e506c796c1b337bc9d7bff945c9a3476fc3c42/core-model/src/main/java/com/sequenceiq/cloudbreak/domain/stack/cluster/DatalakeResources.java#L50-L52
But I can accept your solution if you prefer this way

Reply:
I did see that, but I prefer to have the Json hidden from the reset of the application. Only the entity needs to know that the field is stored as text. Also, I'd rather have any deserialization exceptions get thrown when loading the entity to ensure that we always have good data in the table. We may miss problems if we deserialize it lazily (and repetitively)